### PR TITLE
WIP: resizes thumbnails and sidebar

### PR DESCRIPTION
**Pull request by luisamiranda**
Reducing the size of the Editor Page Sidebar to give more screen area to the asset currently being tagged. First part of Sidebar restyling.

[AB#17652](https://dev.azure.com/dwrdev/web/wi.aspx?pcguid=80f9922c-b416-454c-92c6-ab7b6867e0b2&id=17652)

### CODE REVIEW

**Comment by luisamiranda** [comment]
> screenshot of before and after for comparison

puppy-v-muffin - VoTT 3_19_2019 4_30_32 PM
puppy-v-muffin - VoTT 3_19_2019 4_28_27 PM

File: src/react/components/pages/editorPage/editorPage.scss [hunk 1]
```diff
@@ -116,11 +112,12 @@
             font-size: 80%;
 
             .asset-filename {
-                max-width: 175px;
+                max-width: 110px;
                 overflow: hidden;
                 text-overflow: ellipsis;
                 display: inline-block;
                 white-space: nowrap;
+                text-align: end;
```
Comment (mydiemho) [inline src/react/components/pages/editorPage/editorPage.scss]: what does this do? can I see a before/after for comparison

File: src/react/components/pages/editorPage/editorPage.scss [hunk 2]
```diff
@@ -51,12 +51,8 @@
         }
     }
 
-    .asset-item:first-child {
-        padding: 1em;
-    }
-
     .asset-item {
-        padding: 0.5em 1em;
+        padding: 0.25em 0.25em;
```
[the remark this answers is not in the document]
Comment (luisamiranda) [reply src/react/components/pages/editorPage/editorPage.scss]: I think it wastes more of the space that we're trying to save, but I'm going to check Tyler's Figma files to see if he had suggested a best way.

File: src/react/components/pages/editorPage/editorPage.scss (hunk 1 above)
Comment (luisamiranda) [reply src/react/components/pages/editorPage/editorPage.scss]: Actually, nothing. Will remove. Thanks.

**Comment by luisamiranda** [comment]
New, even smaller slide viewer, following Tyler's specs exactly:

puppy-v-muffin - VoTT 3_21_2019 3_22_27 PM

**Pull request merged**